requirements.txt

```diff
@@ -1,8 +1,7 @@
 pyparsing<3.0.0  # pin for aodhclient which is held for py35
 async_generator
 kubernetes<18.0.0; python_version < '3.6' # pined, as juju uses kubernetes
-# pinned until 3.0 regressions are handled: https://github.com/openstack-charmers/zaza/issues/545
-juju<3.0
+juju
```

Comment:
Don't we want some level of upper-bound restriction? juju<3.2 would pin us to 3.1 compatibility rather than creating a transient break at some later release point for python-libjuju

Reply:
Done